lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -126,6 +129,8 @@
 # to 0 if you are raising the major API version
 LIBPATCH = 17
 
+PYDEPS = ["opentelemetry-api"]
```

Comment:
I think it would be good to call this out in the doc as well. In the ideal case, charmers don't look at the source, and PYDEPS is not necessarily read by any tools, so it's easy to miss in other cases. (Although if they have recent ops, then they'll be fine, but we probably can't assume that yet).